src/docker/docker-compose.yml

```diff
@@ -5,11 +5,15 @@ services:
   # - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@postgres_isolated:5435/tests
   # - TEST_MYSQL_URI=mysql://prisma:prisma@mysql:3306/tests
   # - TEST_MARIADB_URI=mysql://prisma:prisma@mariadb:4306/tests
+  # - TEST_MSSQL_URI=sqlserver://SA:Prisma1-prisma@mssql:1433/master
+  # - TEST_MSSQL_JDBC_URI=sqlserver://mssql:1433;database=master;user=SA;password=Prisma1-prisma;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT
   # Outside Docker
   # - TEST_POSTGRES_URI=postgres://prisma:prisma@localhost:5432/tests
   # - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@localhost:5435/tests
   # - TEST_MYSQL_URI=mysql://prisma:prisma@localhost:3306/tests
   # - TEST_MARIADB_URI=mysql://prisma:prisma@localhost:4306/tests
+  # - TEST_MSSQL_URI=sqlserver://SA:Prisma1-prisma@localhost:1433/master
+  # - TEST_MSSQL_JDBC_URI=sqlserver://mssql:1433;database=master;user=SA;password=Prisma1-prisma;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT
```

Comment:
We need two separate URIs because the package `mssql` and Prisma support different formats for the MSSQL connection string. Once SDK's `uriToCredentials` function supports MSSQL, this will go away.